Add device

holywyvern opened the pull request:
# Summary

- Added Devise as a Gem
- Followed Devise's post install instructions
- Generated the User model
- Added Devise properties into User model
- Added tests for user Creation

# Trello Card

https://trello.com/c/cqsjBZzO

epintos commented:
@rominariedel Don't forget to add the description of the PR

epintos commented on app/views/comments/_form.html.slim:
```diff
@@ -1,11 +1,11 @@
 = form_for([@article, @article.comments.build]) do |f|
   p
-    = f.label :commenter
+    = f.label I18n.t('comment.commenter')
```
Just using `t('comment.commenter')` will work. The I18n is not necessary

Commit:
Added rubocop to project

# Summary

- Added rubocop to Gemfile
- Created .rubocop.yml configuration

# Trello card

https://trello.com/c/D8K0qyQh/8-configurar-rubocop

Commit:
Modified files according rubocop warnings

# Summary

- Added frozen string magical comment to files

Commit:
Added an empty travis-ci config

Commit:
Added a comment to trigger Travis to check this Repo

Commit:
Added some info into README.md
Added the badge and some dependencies

Commit:
Changed ruby requirement from 2.4 to 2.3 in Travis-CI

Commit:
Put 2.3.1 instead of 2.3.0 as ruby version

Commit:
Added bundle install as part of the script

Commit:
Added rspec as a gem dependency

Added rspec to Gemfile for testing

Commit:
Added scss_lint as a gem dependency

Commit:
Added scss lint configuration

Commit:
Installed devise as a gem dependency

Commit:
Added devise's suggestions

Commit:
Added home controller with index action

Commit:
Added user model

Commit:
Added RSpec with some tests for checking than the current user can be created

Commit:
Finished example test

Commit:
Added devise secret key

Commit:
Added magic string to files

Commit:
Fixed rubicop tests

Commit:
Changed rubocop version

juanisierra commented on training-app/app/controllers/home_controller.rb:
```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
```
Remove comments please

juanisierra replied:
It's ok to disable the frozen string literal validation on rubocop using 
```
Style/FrozenStringLiteralComment:
  Enabled: false
```

juanisierra commented on training-app/app/controllers/home_controller.rb:
```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+class HomeController < ApplicationController
+  def index() end
```
Why do you need this controller?

juanisierra commented on training-app/app/helpers/home_helper.rb:
```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
```
Why do you need this helper?

juanisierra commented on training-app/app/models/user.rb:
```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
```
Remove comments please

juanisierra commented on training-app/app/views/home/index.html.erb:
```diff
@@ -0,0 +1 @@
+Hello world
```
Remove unused page please

juanisierra commented on training-app/db/migrate/20180627202539_add_devise_to_users.rb:
```diff
@@ -0,0 +1,51 @@
+      t.inet     :current_sign_in_ip
+      t.inet     :last_sign_in_ip
+
+      ## Confirmable
```
Remove this unused code please

juanisierra commented on training-app/spec/controllers/home_controller_spec.rb:
```diff
@@ -0,0 +1,3 @@
+# frozen_string_literal: true
```
Remove unused specs

juanisierra commented on training-app/spec/helpers/home_helper_spec.rb:
```diff
@@ -0,0 +1,3 @@
+# frozen_string_literal: true
```
Same

Commit:
Removed frozen_string_literal in Rubocop and files

Commit:
Removed home page

Commit:
removed home layout

Commit:
Removed unused fields

Commit:
Removed device routes

Commit:
removed commented code

Commit:
Fixed merge conflicts

Commit:
removed unused commented gems

Commit:
Removed frozen literal string magic comments
